proto/streamlit/proto/openmetrics_data_model.proto

```diff
@@ -1,6 +1,22 @@
+/**!
```

Comment:
I think this file is also not from us: see the original [here](https://github.com/OpenObservability/OpenMetrics/blob/main/proto/openmetrics_data_model.proto)

Reply:
Good point. Added to the exclusion list.

Reply:
https://github.com/streamlit/streamlit/blob/87cb7bc507bbd7a741fab27f5335109363f5d639/scripts/check_license_headers.py#L56